roles/core/precheck/tasks/prepare.yml

```diff
@@ -32,6 +32,17 @@
         line: PubkeyAuthentication yes
         state: present
       notify: reload-sshd
+
+    # prepare | Update ListenAddress at sshd_config for daemon network
+    - name: Update ListenAddress at /etc/ssh/sshd_config
```

Comment:
I'd suggest to follow this convention when naming tasks:
```
- name: prepare | What does the task do?
```

- In this example, `prepare` is the name of the file, as well as the name of the associated tag.
- `What does the task do?` is a high-level description of what this task enables. It's not so relevant which particular option in which particular file is being modified... but what's the impact of this change? i.e. what does the `ListenAddress` option control and why do we want to set a certain value? Examples: "Enable SSH root login" or "Enable SSH pubkey authentication"...